FileManipulator/FileManipulatorModule.cs

```diff
@@ -0,0 +1,11 @@
+﻿using Autofac;
+
+namespace Messerli.FileManipulator
+{
+    public class FileManipulatorModule : Module
+    {
+        protected override void Load(ContainerBuilder builder)
+        {
```

Comment:
Register types

Reply:
https://github.com/messerli-informatik-ag/meta-generator/pull/14/commits/b3e4c85cfd65d3a309eee42308c2d189ef616b4d and https://github.com/messerli-informatik-ag/meta-generator/pull/14/commits/e106256b674f3a6c3e1025196749d5ba00a3890e